pwa-devdocs/src/peregrine/reference/hooks/useEventListener/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: useEventListener
+---
+
+<!--
+The reference doc content is generated automatically from the source code.
+To update this section, update the doc blocks in the source code
+-->
+
+{% include auto-generated/peregrine/src/hooks/useEventListener.md %}
+
+[useQueryResult]: {{site.baseurl}}{%link peregrine/reference/hooks/useQueryResult/index.md %}
```

Comment:
Does this have to be included here because it is referenced by `useEventListener.md` ?

If so, that's gonna be hard for folks to know about ahead of time.